build.assets/Dockerfile-multiarch

```diff
@@ -50,25 +26,213 @@ RUN groupadd ci --gid=$GID -o && \
 
 RUN install --directory --mode=0700 --owner=ci --group=ci /var/lib/teleport
 
-ENV LANGUAGE=en_US.UTF-8 \
-    LANG=en_US.UTF-8 \
-    LC_ALL=en_US.UTF-8 \
-    LC_CTYPE=en_US.UTF-8
-
-RUN yum update -y $YUM_UPDATE_FLAGS && \
-    yum install -y     \
-        git \
-        gcc \
-        gcc-c++ \
-        make \
-        pam-devel \
-        perl-IPC-Cmd \
-        tree \
-        which \
-        zip \
-        zlib-static && \
-    yum clean all && \
-    localedef -c -i en_US -f UTF-8 en_US.UTF-8
+## LIBPCSCLITE ################################################################
+# 
+FROM gcc AS libpcsclite
+ARG LIBPCSCLITE_VERSION
+
+ENV CFLAGS="-std=gnu99"
```

Comment:
Why is this flag needed? I'd expect this to be set by  `configure`.

Reply:
Yes, but on ARM it sets the wrong value for some reason. Had to override it here.

Reply:
Can you please add a comment that this is needed on ARM? I see someone testing it on x86 and removing the comment as "everything works".

Reply:
Yep, already done.